### PR TITLE
Add section detailing how to install ff.vim with other plugin managers

### PR DESCRIPTION
**Pull request by faceleg**
See https://github.com/Shougo/dein.vim/issues/74#issuecomment-208141136

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +224,16 @@ endfunction
 autocmd! User FzfStatusLine call <SID>fzf_statusline()
 ```
 
+Installation Tips
+-----------------
+
+### dein.vim
+
+```
+call dein#add('junegunn/fzf', { 'build': './install', 'merged': 0 })
```
Comment (wsdjeg) [inline README.md]: this only works for unix os,maybe windows has different stuff

Comment (faceleg) [inline README.md]: Agree, hopefully if this gets merged a kind windows user will come along and update this section

Comment (kdelchev) [inline README.md]: README has this line "Also note that fzf currently does not compile on Windows."

**Comment by faceleg** [comment]
@junegunn bump?

**Comment by junegunn** [comment]
fzf and fzf.vim follow standard pathogen-compatible directory layout and should be installable without any special instructions. For example for Vundle, they can be specified just like any other ordinary Vim plugins:

```vim
Plugin 'junegunn/fzf'
Plugin 'junegunn/fzf.vim'
```

For vim-plug it's just:

```vim
Plug 'junegunn/fzf'
Plug 'junegunn/fzf.vim'
```

The README page suggests `dir` and `do` options but they are in fact optional as the plugin automatically downloads the binary if it's not found on the system.

I believe the same goes for NeoBundle, VAM, etc. They're all similar. To be honest, I do not know well enough about dein and its `merged` option you're suggesting here. If dein requires some special configuration for a plugin that other plugin managers don't have any issue with, it is a problem dein has to address.

**Comment by faceleg** [comment]
Thanks!

**Comment by junegunn** [comment]
No problem, thank you for your interest in the project.

**Pull request closed without merging**